release notes/v0.57.0.md

```diff
@@ -0,0 +1,180 @@
+k6 `v0.57.0` is here 🎉! This release includes:
+
+- Introducing helpers for functional testing.
+- The `k6/experimental/csv` module gets a new `asObjects` option.
+- We no longer support the `k6/experimental/browser` module.
+- Moving all non-public APIs to the `internal` package.
+- Many other improvements, bug fixes, and internal refactors.
```

Comment:
This sounds useless, we are highlighting things here.

Reply:
Removed.